src/items/class-features/sow_discontent_(ex).json

```diff
@@ -35,7 +35,7 @@
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<div class=\"page\">\n<p>As a move action, you can attempt to deceive an enemy within 30 feet who can hear you into suspecting its allies of treachery. Attempt a Bluff check with the same DC as a check to feint against that enemy (though this isn’t a standard check to feint, so @UUID[Compendium.sfrpg.feats.11cfGGiQ9oTi4f9G]{Improved Feint (Combat)} and @UUID[Compendium.sfrpg.feats.YL1fs0mA5Gpb7sdV]{Greater Feint (Combat)} don’t apply). If you succeed, the target acts as though they have no allies and are not considered to be an ally to any other creature until the beginning of your next turn. An affected target can’t move freely through former allies’ spaces or flank creatures with them; the target can’t give or receive benefits from the aid another action, any spells, or effects that affect only allies. In addition, if the target can make an attack of opportunity, they do so even against their former allies, but they also avoid actions that would provoke attacks of opportunities from their former allies (even if those creatures wouldn’t make the attack of opportunity). This doesn’t otherwise grant you influence over the affected target or make it more positively disposed toward you or your allies.</p>\n<p>A creature who tries to cast a spell that has a range of touch on the affected target must succeed at an attack roll to touch the target, even if the spell is harmless, though the affected target isn’t forced to attempt saving throws against harmless effects.</p>\n<p>At 6th level, you can spend 1 Resolve Point when you succeed at the Bluff check to have the effect last for a number of rounds equal to your Charisma modifier.&nbsp;</p>\n</div>\n<p><span>&nbsp;</span>@Icon[type:language-dependent] <span>&nbsp;</span><sub><img style=\"user-select: text;\" src=\"systems/sfrpg/images/cup/gameplay/mind.webp\" alt=\"\" width=\"25\" height=\"25\"></sub><span>&nbsp;</span><sub><img style=\"user-select: text;\" src=\"systems/sfrpg/images/cup/gameplay/sense.webp\" alt=\"\" width=\"25\" height=\"25\"></sub></p>\n<p><sub>From AP #35 pg. 51</sub></p>"
```

Comment:
There's still old image elements on here not converted to `@Icon`. Also clear out the `div` wrapping the entire thing